pom.xml

```diff
@@ -119,7 +119,7 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>2.7.17</version>
+      <version>2.18.0</version>
```

Comment:
Not for this PR, but this might be a good first project to start testing out the use of BOMs. We could start by using the google-cloud-java BOM. In fact, let me file an issue on that.